front/src/styles/scss/applications/stdcmV2/_home.scss

```diff
@@ -86,5 +105,147 @@
             height: calc(100vh - 64px);
         }
     }
+    .stdcm-v2-results {
+        background-color: rgb(233, 239, 242);
+        .simuation-banner {
+            color: rgb(0, 0, 0);
+            font-weight: 600;
+            font-size: 1.125rem;
+            line-height: 1.5rem;
+            padding-top: 1.813rem;
+            margin-left: 2rem;
+            margin-right: 2rem;
+            border-bottom: 0.063rem solid rgb(182, 178, 175);
+            .creation-date {
+                color: rgb(121, 118, 113);
+                font-weight: 400;
+                font-size: 0.875rem;
+                line-height: 1.25rem;
+                padding-top: 0.25rem;
+                padding-bottom: 0.688rem;
+            }
+            .simulation-validated {
+                display: flex;
+                .check-circle{
+                  color: rgb(60, 202, 128);
+                  padding-left: 0.5rem;
+                }
+            }   
+        }
+        .simuation-results {
+            display: flex;
+            justify-content: space-between; 
 
+            .table-container {
+                margin-top: 2.125rem;
+                margin-left: 2rem;
+                width: 50.25rem;
+            }
+              .table-results {
```

Comment:
Nit: indentation is a bit funny here (below as well)

Reply:
fix